src/commands/scaling/addScaleRule/queue/QueueAuthSecretStep.ts

```diff
@@ -6,16 +6,24 @@
 import { Secret } from '@azure/arm-appcontainers';
 import { AzureWizardPromptStep, nonNullProp } from '@microsoft/vscode-azext-utils';
 import { QuickPickItem } from 'vscode';
+import { localize } from '../../../../utils/localize';
 import { IAddScaleRuleWizardContext } from '../IAddScaleRuleWizardContext';
 
 export class QueueAuthSecretStep extends AzureWizardPromptStep<IAddScaleRuleWizardContext> {
     public async prompt(context: IAddScaleRuleWizardContext): Promise<void> {
+        const noSecrets: string = localize('noSecretsFound', 'No secrets were found. Create a secret to proceed.');
+        const placeHolder: string = localize('chooseSecretRef', 'Choose a secret reference');
         const containerAppWithSecrets = await context.containerApp.getContainerEnvelopeWithSecrets(context);
         const secrets: Secret[] | undefined = containerAppWithSecrets.configuration.secrets;
```

Comment:
Couldn't you just check to see if `secrets.length` were 0 here and throw an error if that's the case?